src/domain/newsletter/SubscribeNewsletterPage.tsx

```diff
@@ -75,9 +77,13 @@ const SubscribeNewsletterPage: React.FC = () => {
               onClose={notificationOnClose}
             />
           )}
-          <TextWithLineBreaks
-            as="p"
-            text={t('newsletter:subscribeNewsletterPage.leadText')}
+          <p
+            dangerouslySetInnerHTML={{
+              __html: t('newsletter:subscribeNewsletterPage.leadText', {
+                openInNewTab: t('common:srOnly.opensInANewTab'),
+                url: PRIVACY_POLICY_LINKS[locale],
+              }),
+            }}
```

Comment:
You might be able to avoid `dangerouslySetInnerHTML` and defining HTML in the translation data by using the `Trans` component:

https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0